prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE >
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+    <body>
+      <script>
+        console.log('I have added JS to a script tag in the HTML!');
+      </script>
+
+      <script src="main.js">
+        console.log(' I have added a script file via the src attribute ');
+      </script>
```

Comment:
For this one, you should not have anything in-between the opening and closing `script` tag. You'll just load in the `main.js` file and your `console.log` will just be in there